encryptonize/defs.go

```diff
@@ -18,12 +18,18 @@ package encryptonize
 //                               Utility                               //
 /////////////////////////////////////////////////////////////////////////
 
+// VersionResponse is the response to a Version call.
 type VersionResponse struct {
+	// Git commit of the current version
 	Commit string `json:"commit"`
-	Tag    string `json:"tag"`
+
+	// Version tag of the current version
+	Tag string `json:"tag"`
 }
 
+// HealthResponse is the response to a Health call.
 type HealthResponse struct {
+	// Current health status of the server.
```

Comment:
Would adding the possible states to the documentation make sense? A string is very general

Reply:
Added a link to the go health check docs.